pkg/metrics/features/cell.go

```diff
@@ -20,6 +21,11 @@ import (
 	"github.com/cilium/cilium/pkg/promise"
 )
 
+var (
+	// withDefaults will set enable all default metrics in the agent.
+	withDefaults = os.Getenv("CILIUM_FEATURE_METRICS_WITH_DEFAULTS")
```

Comment:
Do we need to define package globals for this? It's only used in one place.

Reply:
True, usually I prefer to keep these type of variables, the ones retrieved from Getenv, at the very top of the file.